### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by SXKDZ**

### CODE REVIEW

File: vj4/locale/zh_CN.yaml [hunk 1]
```diff
@@ -382,8 +381,8 @@ Delete Selected User: 删除所选用户
 Set Roles for Selected User: 设置所选用户的角色
 Create Role: 创建角色
 Role name can only contains numbers, letters and underscores.: 角色名称只能包含数字、字母和下划线。
-'Built-in: Effective for visitors that not logged in': '内置: 对未登录的游客有效'
-'Built-in: Effective for registered users that not a member of the domain': '内置: 对域外注册用户有效'
+'Built-in: For visitors that are not logged in': '内置: 对未登录的游客有效'
```
Comment (breeswish) [inline vj4/locale/zh_CN.yaml]: The meaning of “有效” is missing

**Comment by breeswish** [comment]
BTW, could you consider replacing all EN translation keys with tokens?

https://github.com/vijos/vj4/issues/90

File: vj4/locale/zh_CN.yaml (hunk 1 above)
Comment (SXKDZ) [inline vj4/locale/zh_CN.yaml]: i think 'for' in enough

Comment (twd2) [inline vj4/locale/zh_CN.yaml]: The meaning of “有效” is missing.

**Pull request merged**